atef/widgets/config/utils.py

```diff
@@ -373,6 +374,16 @@ def add_with_range():
             )
             self.suggest_comparison.emit(comparison)
 
+        def open_arch_viewer():
+            arch_widget = get_archive_viewer()
+            for datum in data:
+                arch_widget.add_signal(datum.pvname)
+            arch_widget.show()
+
+        menu.addSection("Open Archive Data viewer")
+        archive_viewer_all = menu.addAction("View all selected")
```

Comment:
I think it's worthwhile to have a prominent additional button between 'select' and 'update data' in the happi component selection dialog.

Reply:
What kind of button are you thinking of?  Right now the archive viewer button will open the viewer widget if it doesn't exist or add pv's the widget if it does.

Reply:
Hmm, maybe I'm missing the button. Could you include a screenshot? This is the context menu where I've been accessing the archive viewer plots from:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/5139267/200675787-b76182f3-1c36-4802-9b84-d9eb79fc48dc.png">
I was thinking a button in between "Select" and "Update data" would be nice

Reply:
Interesting.  For me it renders like this, with context menu section headers:
![image](https://user-images.githubusercontent.com/35379409/200694757-33d66aa7-f0f2-43df-b07f-8939e9504d36.png)

But I realize now you're thinking of a button outside of the context menu that will open the archive viewer.  That's something I can add 👍

Reply:
Yeah, those section headers are OS dependent, frustratingly enough.